src/gwt/src/org/rstudio/studio/client/rsconnect/ui/NewRSConnectAuthPage.java

```diff
@@ -305,14 +305,17 @@ public void execute(NewRSConnectAccountResult input)
                   contents_.showWaiting();
                   
                   // prepare a new window with the auth URL loaded
-                  NewWindowOptions options = new NewWindowOptions();
-                  options.setName(AUTH_WINDOW_NAME);
-                  options.setAllowExternalNavigation(true);
-                  options.setShowDesktopToolbar(false);
-                  display_.openWebMinimalWindow(
-                        result_.getPreAuthToken().getClaimUrl(),
-                        false, 
-                        700, 800, options);
+                  if (canSpawnAuthenticationWindow())
```

Comment:
Currently, this UI is displayed (with the link to click, opening in an external browser by default):

<img width="431" alt="screen shot 2015-10-26 at 6 20 29 pm" src="https://cloud.githubusercontent.com/assets/1976582/10747063/50afcdca-7c0e-11e5-916c-057a5d0fd1b9.png">

We should probably change the UI in this case (since we aren't actually making any effort to open a link).